sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/samples/Sample1_HelloWorld.cs

```diff
@@ -8,66 +8,9 @@
 using Azure.Core.TestFramework;
 using Azure.Analytics.Synapse.Tests;
 using Azure.Analytics.Synapse.AccessControl;
-using Azure.Analytics.Synapse.AccessControl.Models;
 using Azure.Identity;
 using NUnit.Framework;
 
 namespace Azure.Analytics.Synapse.AccessControl.Samples
 {
-    /// <summary>
```

Comment:
Samples will be restored in: https://github.com/azure/azure-sdk-for-net/issues/24395